extensions/clickup/package.json

```
@@ -1,18 +1,26 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "manage-clickup-tasks",
-  "title": "Clickup Tasks Explorer",
+  "title": "Tasks Explorer",
```

```suggestion
  "title": "ClickUp - Tasks Explorer",
```
Maybe this is a better title?